client/src/pages/Location.js

```diff
@@ -1,17 +1,16 @@
 import React from "react";
 import styled from "styled-components/macro";
 import PageHeadline from "../components/Header/PageHeadline";
-import placeholder from "../assets/placeholder.jpeg";
-import ImageCard from "../components/Card/ImageCard";
-import mappath from "../assets/map.svg";
+// import placeholder from "../assets/placeholder.jpeg";
+// import ImageCard from "../components/Card/ImageCard";
 
-import { Link, useParams } from "react-router-dom";
+import { useParams } from "react-router-dom";
 import { useQuery } from "react-query";
```

Comment:
Maybe you could use also `QueryCache, ReactQueryCacheProvider`

See Example:
- https://react-query.tanstack.com/docs/examples/simple

---

There is also a great dev tool for the react-query library:
- https://react-query.tanstack.com/docs/devtools